packages/icons/icon-data.ts

```diff
@@ -0,0 +1,487 @@
+export type IconType =
```

Comment:
Looks like this file is generated. Please add a comment at the top to warn people don't modify this file directly, but run a command to generate new data.